src/local/document.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+
+// The local Cozy Desktop metadata, as stored in PouchDB
```

Comment:
I thought of pouch as being part of "local", since it's mostly used to implement local metadata storage.
But I don't have a clear view of the reconciliation part and the limit between it and "local".
So ok to rename `local/document` to `document` if it makes more sense.